### DISCUSSION TITLE
$set TextInput value outside a Repeater Component

### DISCUSSION THREAD

**Discussion by tangjh** [filamentphp/filament#1177] [Help] [answered]
Is it possible to set a TextInput field that is outside of a repeater in the same form?
I tried $set but it is for that repeat component.
I tried $livewire->data and the data is updated but it doesn't update the UI TextInput field.

**Comment by danharrin** [comment, marked as answer]
Hey, you can `$set` or `$get` outside of a repeater component by setting the final parameter to be `true`. This enables "absolute" state paths instead of those relative to the current component.

Imagine we had a component with the state path `data.repeater.item1.first_name`:
- `$get('last_name')` would return the value of `data.repeater.item1.last_name`
- `$get('data.title', true)` would return the value of `data.title`, not `data.repeater.item1.data.title`

So in your case, you probably want something like `$set('data.FIELD_NAME', $value, true)`.

**Reply by tangjh** [reply]
Thanks!  I could have miss the documentation.
